### DISCUSSION TITLE
You can set this permanently in your config using mpf:report_crashes: "never" or "always"

### DISCUSSION THREAD

**Discussion by worldpeace-germany** [missionpinball/missionpinball#81] [Q&A] [answered]
Hi,

what config is meant here, my config.yaml? It tells me it neither knows mpf:report_crashes nor report_crashes. 

What do I have to enter?

Thanks!

**Comment by Pucksnok** [comment, marked as answer]
I use it in my config.yaml, seems to work for me.


```
 ​#​config_version=5
 
 ​mpf​:
 ​    ​report_crashes​: ​never
```
